addon/components/materialize-copyright.js

```diff
@@ -0,0 +1,26 @@
+import Ember from 'ember';
+import layout from '../templates/components/materialize-copyright';
+
+export default Ember.Component.extend({
+  init: function() {
+    this._super(...arguments);
+    if (this.get('startYear') !== null) {
```

Comment:
This conditional is really just another part of your assert condition. You should put it in the assert its self